README.md

```diff
@@ -113,24 +467,67 @@ This example branch can serve as the basis for what you might want to provide fo
 
 4. You should now be able to see push notifications coming across the local network.
 
-### Update the Android example app to send push notifications
+### Update the Android quickstart app to send push notifications
 
 1. Checkout the `push-notifications-example` branch
 
 2. Add your `google-services.json` file to the `example/android/app` folder if you haven't already done it as a part of the FCM project creation process.
 
-2. Uncomment `apply plugin: 'com.google.gms.google-services'` in the example app's `build.gradle` file.
+3. Uncomment `apply plugin: 'com.google.gms.google-services'` in the quickstart app's `build.gradle` file.
 
-3. Uncomment `classpath('com.google.gms:google-services:4.3.15')` in the top level of the example app's `build.gradle` file.
+4. Uncomment `classpath('com.google.gms:google-services:4.3.15')` in the top level of the quickstart app's `build.gradle` file.
 
-4. Sync the gradle project.
+5. Sync the gradle project.
 
-5. Replace `YOUR_SERVER_ADDRESS` in the `PullController.ts` file. If you're using the example notification server, it should be something like `YOURIPADDRESS:8080` since the Android emulator takes over localhost.
+6. Replace `YOUR_SERVER_ADDRESS` in the `PullController.ts` file. If you're using the example notification server, it should be something like `YOURIPADDRESS:8080` since the Android emulator takes over localhost.
 
-6. Change the example app's environment to `production` in both places in `AuthView.tsx`.
+7. Change the quickstart app's environment to `production` in both places in `AuthView.tsx`.
 
-7. Replace `YOUR_FIREBASE_SENDER_ID` in the `PullController.ts` with your sender ID from Firebase.
+8. Replace `YOUR_FIREBASE_SENDER_ID` in the `PullController.ts` with your sender ID from Firebase.
 
-### Update the iOS example app to send push notifications
+### Update the iOS quickstart app to send push notifications
+
+<!--any thoughts on providing more details here?-->
```

Comment:
See comment

Reply:
On the branch iOS pushes should just work.